src/terrain/clients/analyses.clj

```diff
@@ -0,0 +1,40 @@
+(ns terrain.clients.analyses
+  (:require [terrain.util.config :refer [analyses-base-uri]]
+            [clj-http.client :as http]
+            [cemerick.url :refer [url]]
+            [terrain.auth.user-attributes :refer [current-user]]
+            [slingshot.slingshot :refer [throw+]]))
+
+
+(defn analyses-url
+  ([components]
+   (analyses-url components {}))
+  ([components query]
+   (-> (apply url (cons (analyses-base-uri) components))
```

Comment:
I didn't catch this before, but `(apply url (cons (analyses-base-url) components))` can be shortened to `(apply url (analyses-base-url) components)`. This is optional, of course. I'm just lazy and I usually use whichever option requires less typing. 😆